# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../eglibc/eglibc-2.10.1-fixes.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Submitted By: Jim Gifford (jim at cross-lfs dot org)
Date: 07-27-2009
Initial Package Version: 2.10.1
Origin: Upstream
Upstream Status: Applied
Description: These are fixes eglibc-2.10.1, and should be
             rechecked periodically.

diff -Naur libc.orig/elf/ldd.bash.in libc/elf/ldd.bash.in
--- ./libc.orig/elf/ldd.bash.in	2009-02-13 18:31:14.000000000 +0000
+++ ./libc/elf/ldd.bash.in	2009-06-07 23:26:18.000000000 +0000
@@ -1,4 +1,4 @@
-#! @BASH@
+#! /bin/bash
 # Copyright (C) 1996-2008, 2009 Free Software Foundation, Inc.
 # This file is part of the GNU C Library.
 
diff -Naur libc.orig/sunrpc/Makefile libc/sunrpc/Makefile
--- ./libc.orig/sunrpc/Makefile	2008-04-25 11:12:05.000000000 +0000
+++ ./libc/sunrpc/Makefile	2009-06-07 23:26:18.000000000 +0000
@@ -144,11 +144,11 @@
 	$(+link)
 
 $(addprefix $(objpfx)cross-,$(rpcgen-objs)): $(objpfx)cross-%.o: %.c
-	gcc $< -c -D_RPC_THREAD_SAFE_ -D_CROSS_RPCGEN_ \
+	$(BUILD_CC) $< -c -D_RPC_THREAD_SAFE_ -D_CROSS_RPCGEN_ \
 		$(OUTPUT_OPTION) $(compile-mkdep-flags)
 
 $(objpfx)cross-rpcgen: $(addprefix $(objpfx)cross-,$(rpcgen-objs))
-	gcc $^ -o $@
+	$(BUILD_CC) $^ -o $@
 
 # This makes sure -DNOT_IN_libc is passed for all these modules.
 cpp-srcs-left := $(rpcgen-objs:.o=.c)
diff -Naur libc.orig/timezone/Makefile libc/timezone/Makefile
--- ./libc.orig/timezone/Makefile	2009-03-03 13:14:09.000000000 +0000
+++ ./libc/timezone/Makefile	2009-06-07 23:26:18.000000000 +0000
@@ -181,11 +181,11 @@
 $(objpfx)zic: $(addprefix $(objpfx), $(zic-objs))
 
 $(addprefix $(objpfx)cross-,$(zic-objs)): $(objpfx)cross-%.o: %.c
-	gcc $< -c $(OUTPUT_OPTION) $(CFLAGS-$*.c) $(CPPFLAGS-$*) \
+	$(BUILD_CC) $< -c $(OUTPUT_OPTION) $(CFLAGS-$*.c) $(CPPFLAGS-$*) \
 		-DCROSS_ZIC $(compile-mkdep-flags)
 
 $(objpfx)cross-zic: $(addprefix $(objpfx)cross-,$(zic-objs))
-	gcc $(addprefix $(objpfx)cross-,$(zic-objs)) -o $@
+	$(BUILD_CC) $(addprefix $(objpfx)cross-,$(zic-objs)) -o $@
 
 tz-cflags = -DTZDIR='"$(zonedir)"' \
 	    -DTZDEFAULT='"$(localtime-file)"' \
